packages/backend/src/app/helper/base-entity.ts

```diff
@@ -1,4 +1,13 @@
 import { EntitySchemaColumnOptions } from 'typeorm'
+import { system } from './system/system'
+import { SystemProp } from './system/system-prop'
+import { ApEnvironment } from '@activepieces/shared'
+
+export const JSON_COLUMN_TYPE = system.get(SystemProp.ENVIRONMENT) === ApEnvironment.DEVELOPMENT ? 'simple-json' : 'json'
+export const JSONB_COLUMN_TYPE = system.get(SystemProp.ENVIRONMENT) === ApEnvironment.DEVELOPMENT ? 'simple-json' : 'jsonb'
+export const BLOB_COLUMN_TYPE = system.get(SystemProp.ENVIRONMENT) === ApEnvironment.DEVELOPMENT ? 'blob' : 'bytea'
+export const TIMESTAMP_COLUMN_TYPE = system.get(SystemProp.ENVIRONMENT) === ApEnvironment.DEVELOPMENT ? 'datetime' : 'timestamp with time zone'
+export const COLLATION = system.get(SystemProp.ENVIRONMENT) === ApEnvironment.DEVELOPMENT ? undefined : 'en_natural'
```

Comment:
It seems like this logic prevents using `sqlite3` in production or testing, right?

Would it be possible to detect the database and adjust data types accordingly?